test/runtests.jl

```diff
@@ -10,26 +10,25 @@ performance = true
 isCI = haskey(ENV, "CI")
 notCI = !isCI
 
-# TODO: switch the JuLIP test to an ASE test
-# check whether we have JuLIP
-# hasjulip = true
-# try
-#    using JuLIP
-# catch
-#    hasjulip = false
-# end
+# Check for CUDA availability (for GPU tests)
```

Comment:
Would be nice to try not just CUDA, e.g. see here: 

https://github.com/ACEsuit/EquivariantTensors.jl/blob/main/test/test_utils/utils_gpu.jl